### DISCUSSION TITLE
Using OneDrive for Business as Backup location

### DISCUSSION THREAD

**Discussion by lombakp** [standardnotes/forum#3033] [General] [answered]
Hi,
I was just wondering if anyone is using OneDrive for Business as a method of backing up Standard Notes. 

I Sync my business one drive to my mac, so, being lazy, I just created a folder in the one drive file space and pointed the automatic 'local' backup to that folder.

So far I haven't had a problem - anyone else using this method. 
Is there a reason I should NOT be using this method?

Thanks for your comments.
Ken

**Comment by effieeee** [comment, marked as answer]
Hi Ken, this is perfectly fine, especially since we've had to disable the direct automatic cloud backups to the 3rd party cloud services! 🙏

**Comment by lombakp** [comment]
Thank you. 
Ken
